scale/storage/management/commands/scale_destroy_file.py

```diff
@@ -0,0 +1,63 @@
+"""Defines the command line method for running a destroy file task"""
+from __future__ import unicode_literals
+
+import logging
+import signal
+import sys
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+
+import storage.destroy_file_job as destroy_file_job
+from messaging.manager import CommandMessageManager
+from storage.messages.delete_files import create_delete_files_messages
+from storage.models import ScaleFile
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Command that executes the destroy file process for a given file
+    """
+
+    help = 'Perform a file destruction operation on a file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('-f', '--file_ids', nargs='+', type=int, required=True,
+                            help='List of file IDs to delete')
+        parser.add_argument('-j', '--job_id', action='store', type=int, required=True,
+                            help='ID of the Job model')
+
+    def handle(self, *args, **options):
+        """See :meth:`django.core.management.base.BaseCommand.handle`.
+
+        This method starts the file destruction process.
+        """
+
+        # Register a listener to handle clean shutdowns
+        signal.signal(signal.SIGTERM, self._onsigterm)
+
+        file_ids = options.get('file_ids')
+        job_id = options.get('job_id')
+
+        logger.info('Command starting: scale_destroy_file')
+        logger.info('File IDs: %s', file_ids)
```

Comment:
We want to make this job (and other new system jobs) not rely on a Scale database connection. We need to pass in all of the meta-data we need for deleting the files, such as what workspace/broker they exist on, file ID, path within the workspace, etc.